src/components/searchSection/catergrayList.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { MDBIcon } from 'mdbreact';
+import './myStyle.scss';
+
+function CatergrayList({ items }) {
+  let itemsTags = [];
+  let numberOfItems = items.length;
+  let [ActiveBubble, setActiveBubble] = useState(0);
+  let [setCatogray] = useState('');
+
+  //light the bubble by pink color
+  const handelBubble = (index) => {
+    if (ActiveBubble === index) {
+      return 'textPink';
+    }
+  };
+
+  const emptyItems = () => {
+    for (let i = 1; i <= 9 - numberOfItems; i++) {
+      itemsTags.push(<div md="12" className="lightBlack h-10 mb-1"></div>);
+    }
+  };
```

Comment:
I couldn't understand what this method do?

Reply:
will create empty div to full it in the catogray items

Reply:
did you have to have an empty divs do that?